BepInEx.Preloader/RuntimeFixes/UnityPatches.cs

```diff
@@ -40,17 +40,13 @@ public static void GetCodeBase(ref string __result, Assembly __instance)
 				__result = $"file://{location.Replace('\\', '/')}";
 		}
 
-#if UNITY_2018
-/*
- * DESC: Workaround for Trace class not working because of missing .config file
- * AFFECTS: Unity 2018+ (not .NET Standard / MonoBleedingEdge runtimes)
- */
 		[HarmonyPostfix, HarmonyPatch(typeof(AppDomain), nameof(AppDomain.SetupInformation), MethodType.Getter)]
 		public static void GetExeConfigName(AppDomainSetup __result)
 		{
+			if (!Preloader.IsDotNet46)
```

Comment:
Could consider patching each method in here (semi) manually in the `Apply()` method instead of doing a blanket `PatchAll()`.

Maybe create a custom attribute specifying which patches are for which Unity version(s), but might not be worth it considering the small amount of patches we currently have

Reply:
In theory this particular patch can be deleted because of fixes in Doorstop.

Reply:
Removed the patch since we can rely on latest Doorstop instead.